overview/templates/overview/candidate_detail.html

```diff
@@ -83,6 +83,24 @@
                 </tr>
             </tr>
             {% endif %}
+
+            {% if endorsements %}
+                <tr>
+                    <th>Endorsements</th>
+                    <td>
+                        {% for endorsement in endorsements %}
+                        <a href="{{ endorsement.link }}" target="_blank" data-toggle="tooltip" title="{{ endorsement.organization.name }} endorsed {{ object.shortname }} on {{ endorsement.date|date }}">
```

Comment:
the link also isn't required but just links back to the page if it isn't present, which seems ok